demo/alloy-showcase-webapp/src/test/java/com/liferay/faces/test/alloy/showcase/datatable/DataTableTesterBase.java

```diff
@@ -238,37 +184,31 @@ protected String getRowSelectionInputXpath(Customer customer) {
 			"')]//input[contains(@id,'customers:')]";
 	}
 
-	/**
-	 * Determines whether or not the specified page number is active in the rendered <code>alloy:paginator</code>
-	 * markup.
-	 */
-	protected ExpectedCondition<Boolean> isPaginatorPageNumberButtonActive(int pageButtonNumber) {
```

Comment:
@stiemannkj1 I realize that not all ExpectedConditions are boolean, but I think that the boolean ones read better with an "is"